src/sentry/runner/commands/run.py

```diff
@@ -540,6 +551,30 @@ def ingest_consumer(consumer_types, all_consumer_types, **options):
     if not all_consumer_types and not consumer_types:
         raise click.ClickException("Need to specify --all-consumer-types or --consumer-type")
 
+    if v2_consumer:
+        from sentry.ingest.consumer_v2.factory import get_ingest_consumer
+
+        if len(consumer_types) != 1:
+            raise click.ClickException(
+                "Only a single `--consumer-type` allowed when using the `v2` ingest consumer"
+            )
+
+        # FIXME: Judging by the default values, `max_batch_time` is in milliseconds,
+        # but arroyo expects that option to be in seconds, so fix that up here:
+        if (mbt := options["max_batch_time"]) > 10:
+            options["max_batch_time"] = mbt / 1000
+
+        topic = ConsumerType.get_topic_name(next(iter(consumer_types)))
+        consumer = get_ingest_consumer(topic=topic, **options)
```

Comment:
You need to call `configure_metrics()` with a metrics backend before this in order for Arroyo to emit any metrics https://getsentry.github.io/arroyo/metrics.html

Reply:
Added in 18c0270

Reply:
now the tests are failing because `configure_metrics` leaks from previous tests. we have no precedent for how to solve this problem, because the arroyo consumers typically test processing strategies instead of entire CLI commands. I would suggest to mock out `configure_metrics` in tests to noop...